libs/sdk-core/src/breez_services.rs

```diff
@@ -425,8 +427,39 @@ impl BreezServices {
     }
 
     pub async fn reverse_swap_info(&self) -> Result<ReverseSwapInfo> {
-        let rsi = crate::boltzswap::reverse_swap_info().await?;
-        return Ok(rsi);
+        self.btc_send_swapper
+            .reverse_swapper_api
```

Comment:
I think it is better to make the reverse_swapper_api private field of the btc_send_swapper and make an appropriate method that uses it internally.

Reply:
Done.